src/components/views/messages/TextualBody.js

```diff
@@ -262,6 +262,7 @@ export default class TextualBody extends React.Component {
 
         // exploit that events are immutable :)
         return (nextProps.mxEvent.getId() !== this.props.mxEvent.getId() ||
+                nextProps.mxEvent !== this.props.mxEvent ||
```

Comment:
Object identity check should be faster than the reading the ID, I'd move this up before the previous line.